File: RummikubWebClient/src/java/client/servlets/Cookie/CookieUtils.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package client.servlets.Cookie;

import javax.servlet.http.Cookie;

/**
 *
 * @author yafita870
 */
public class CookieUtils {
    
    public static String CookieMap(Cookie[] cookis ,String key )
    {
        for (Cookie cooki : cookis) {
            String str = cooki.getName();
            if(str.equals(key))
            {
                return cooki.getValue();
            }
        }
        
        return "error";
    }
}

File: RummikubWebClient/web/js/CookieUtils.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */


function setCookie(cookieName, cookieValue){
    document.cookie = cookieName + "=" + cookieValue +"; path=/";
}

function getCookie(cookieName){
    var name = cookieName + "=";
    var cookies = document.cookie.split(';');

    for(var i = 0; i < cookies.length; ++i){
        var cookie = cookies[i];

        while(cookie.charAt(0) == ' '){
            cookie = cookie.substring(1);
        }

        if(cookie.indexOf(name) == 0){
            return cookie.substring(name.length, cookie.length);
        }
    }

    return "";
}